microsoft/atlas-design
# [Draft] Tokens Page Redesign

Task: task-[work-item-number]

Link:

https://www.figma.com/file/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?node-id=2%3A0

Redesigning the tokens page

## Testing

1. checkout aakers/tokens and navigate to sites/
2.  run `npm run start`
3. Navigate to [tokens](http://localhost:1111/tokens/colors.html)
   Expected result: You should see the tokens page aligning with the Figma design provided

## Additional information

- Need to figure out why styles are not being applied during dynamic rendering of the color divs(ex. background-color).
